packages/core/src/entity/EntityAssigner.ts

```diff
@@ -325,7 +333,7 @@ export interface AssignOptions<Convert extends boolean> {
   onlyProperties?: boolean;
 
   /**
-   * With `onlyOwnProperties` enabled, to-many relations are skipped, and payloads of to-one relations are converted
+   * With `onlyOwnProperties` enabled, (not owned) to-many relations are skipped, and payloads of other relations are converted
```

Comment:
```suggestion
   * With `onlyOwnProperties` enabled, inverse sides of to-many relations are skipped, and payloads of other relations are converted
```

also let's also update the docs here:

https://mikro-orm.io/docs/entity-helper#assign-options